packages/keystatic/src/form/api.tsx

```diff
@@ -451,6 +452,8 @@ export function component<
     schema: Schema;
     /** The label to show in the insert menu and chrome around the block if chromeless is false */
     label: string;
+    /** An icon to show in the toolbar for this component block. Component blocks with `toolbarIcon` are shown in the toolbar directly instead of the insert menu */
```

Comment:
> instead of the insert menu

Are component blocks with `toolbarIcon` omitted from the insert menu?

Reply:
Yes, it's slightly confusing since by "insert menu" here, I mean menu on the plus button in the toolbar, not the `/` menu, they still appear in the `/` menu (as do many other things that aren't in the other menu in the toolbar)